http-custom-mechanism/custom-module/add-custom-module.cli

```diff
@@ -1,3 +1,3 @@
 # Add the custom module to the server
 
-module add --name=org.jboss.as.quickstart.http_custom_mechanism.custom-http-mechanism --resources=target/http-custom-mechanism.jar --dependencies=org.wildfly.security.elytron,javax.api
+module add --name=org.jboss.as.quickstart.http_custom_mechanism.custom-http-mechanism --resources=target/custom-module.jar --dependencies=org.wildfly.security.elytron,javax.api
```

Comment:
Why is this being renamed?

Reply:
Actually, just noticed that you already mentioned this in the description so you can ignore this comment :)